client/beefy/src/lib.rs

```diff
@@ -40,6 +40,12 @@ mod worker;
 
 pub mod notification;
 
+/// payload type.
+pub type Payload = Vec<(BeefyPayloadId, Vec<u8>)>;
+
+/// Id of the mmr root hash in the [`Commitment`] payload
+pub const MMR_ROOT_ID: BeefyPayloadId = *b"mh";
```

Comment:
Could you move these things to `primitives/beefy`? I think they would be useful in general, especially the constant.

The Payload ids should be grouped into a module like:
```rust
pub mod known_payload_ids {
  pub const MMR_ROOT_HASH: BeefyPayloadId = b"mh";
}
```